signer/tests/integration/transaction_coordinator.rs

```diff
@@ -2134,13 +2137,13 @@ async fn sign_bitcoin_transaction() {
     let amount = 2_500_000;
     let signers_public_key = shares.aggregate_key.into();
     let max_fee = amount / 2;
-    let (deposit_tx, deposit_request, _) =
+    let (deposit_tx, deposit_request, deposit_info) =
         make_deposit_request(&depositor, amount, utxo, max_fee, signers_public_key);
     rpc.send_raw_transaction(&deposit_tx).unwrap();
 
     assert_eq!(deposit_tx.compute_txid(), deposit_request.outpoint.txid);
 
-    let body = deposit_request.as_emily_request(&deposit_tx);
+    let body = deposit_request.as_emily_request(&deposit_tx, deposit_info.reclaim_script);
```

Comment:
It feels more natural to move the `as_emily_request` method to the `DepositInfo` struct, since that has everything we need (besides the raw transaction) and we always have one available when we want to make an Emily request.